friends/src/FriendsList.js

```diff
@@ -0,0 +1,163 @@
+import React from 'react';
+import axios from 'axios';
+import Friend from './Friend';
+import UpdateFriend from './UpdateFriend';
+import FriendListHeader from './FriendListHeader';
+import { ListWrapper, StyledForm } from './Styled';
+
+class FriendsList extends React.Component {
+  constructor(props) {
```

Comment:
Referenced this in our team channel, give it shot.
```suggestion
state = {
   friends: [],
   name: '',
   age: '',
   email: '',
   id: ''
};
```